db/api/models/index.js

```diff
@@ -1,6 +1,30 @@
+const { response } = require("express");
 const db = require("../../connection");
+
 exports.selectCategories = () => {
   return db.query(`SELECT * FROM categories`).then((response) => {
     return response.rows;
   });
 };
+
+exports.selectReviews = () => {
+  return db
+    .query(
+      `SELECT  title,
+  designer,
+  owner,
+  review_id,
+  review_img_url ,
+  category,
+  created_at,
+  votes, 
+  COALESCE((SELECT COUNT(review_id)
+FROM comments
+WHERE review_id = reviews.review_id),0) AS comment_count
+  FROM reviews
+  ORDER BY created_at DESC`
+    )
+    .then((response) => {
+      return response.rows;
```

Comment:
It seems to work, so im hestitate to get you to change it, but ideally you'd do a JOIN to join the reviews and comments, then do a count on that.